common/source/docs/common-electro-permanent-magnet-V3.rst

```diff
@@ -17,7 +17,7 @@ Objects that it grips onto should be metal or have the included `thin steel targ
 attached to them.
 
 The device accepts a standard PWM servo input or UAVCAN and uses a 5V power supply.  
-Significant amounts of power are only required for about about 1 second when it first grips or releases the cargo.
+Significant amounts of power is only required for about 1 second when it first grips or releases the cargo.
```

Comment:
"are" is correct when you refer to plural of something - so "amount of power is" or "amounts of power are". I suggest perhaps "A significant amount of power is"